module-generation/scrape.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+import requests
+from github import Github
+
+indexDir = "./scrape-data/"
+os.mkdir(indexDir)
+
+accessToken = os.environ.get('GIT_TOKEN')
+g = Github(accessToken)
+
+user = g.get_user('Terasology')
+repo = user.get_repo("Index")
+allModules = repo.get_contents("/distros/omega/gradle.properties")
+getModules = allModules.decoded_content.decode()
+for line in getModules.split():
+    if  line.startswith('extraModules='):
+        filterData=line[13:]
+        modules=filterData.split(",")
+
+# Collect modules 
+for module in range(len(modules)):
+    if(module<len(modules)):
+        repository=user.get_repo(modules[module])
+
+        #Fetch module information
+        try:
+            moduleContent=repository.get_contents("module.txt")
+            getModuleData=moduleContent.decoded_content.decode()
+            parseModuleData=json.loads(getModuleData)
```

Comment:
again, those are good names for methods, but not for variables.
method names are typically actions that indicate what is done in the method.
variable names should be passive descriptions of what information they store.

Reply:
done